lib/ui/painting/canvas.h

```diff
@@ -106,12 +106,12 @@ class Canvas : public RefCountedDartWrappable<Canvas> {
   void drawPath(const CanvasPath* path,
                 const Paint& paint,
                 const PaintData& paint_data);
-  void drawImage(const CanvasImage* image,
+  void drawImage(CanvasImage* image,
```

Comment:
The reason to drop const is `GetAllocationSize` is _not_ const. Filed https://github.com/flutter/flutter/issues/58422 to investigate making it const and adding the qualifiers back here.

Reply:
Looks like you just have to add the const qualifier to GetAllocationSize.  I checked out the implementations and didn't see any mutations.

Reply:
Opened https://github.com/flutter/engine/pull/18713 so that doesn't get reverted if this does.  I'll merge that one into this.